### PR TITLE
fix(transition guides): removing old transition guides

### PR DESCRIPTION
**Pull request by homelessbirds**
[NR-67634]

### CODE REVIEW

File: src/content/docs/new-relic-solutions/get-started/intro-new-relic.mdx [hunk 1]
```diff
@@ -20,6 +20,7 @@ redirects:
   - /docs/using-new-relic/
   - /docs/using-new-relic/welcome-new-relic
   - /docs/using-new-relic/welcome-new-relic/get-started/intro-new-relic
+  - docs.newrelic.com/docs/new-relic-solutions/new-relic-one/ui-data/new-relic-feature-end-life-announcements-july-2020
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/get-started/intro-new-relic.mdx:23]: Should this start with the /docs/ portion?

File: src/content/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards.mdx [hunk 2]
```diff
@@ -51,6 +51,7 @@ redirects:
   - /docs/insights/new-relic-insights/understanding-insights/new-relic-insights
   - /docs/insights/using-insights-ui/getting-started/introduction-new-relic-insights
   - /docs/insights/new-relic-insights/decorating-events/insights-attributes
+  - docs/new-relic-solutions/new-relic-one/core-concepts/transition-new-relic-one-insights/
```
Comment (jeff-colucci) [inline src/content/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards.mdx:54]: / before the docs/ here I think, also I'm not sure but I think we can remove the last / but I don't know if it matters or not

**Review by jeff-colucci — DISMISSED** [review]
Only a couple small things on the back end, other than that looks great!

File: src/content/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards.mdx]: done! thanks!

File: src/content/docs/new-relic-solutions/get-started/intro-new-relic.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/get-started/intro-new-relic.mdx]: done! thanks!

**Review by jeff-colucci — APPROVED** [review]
Looks great!

**Pull request merged**